test/unit/lifecycle/properties.js

```diff
@@ -29,35 +29,35 @@ describe('lifecycle/property', function () {
   it('should return an function', function () {
     expect(propertiesInit()).to.be.a('function');
   });
-  
+
   it('should not leak options to other definitions', function () {
     const elem = element().skate({
       properties: {
-        test1: { 
-          attribute: true, 
-          default: 'test1', 
+        test1: {
+          attribute: true,
+          default: 'test1',
```

Comment:
`default` is only used if the value, or initial value is "empty" (`null` or `undefined`). This also means that `default` can be used not only to initialise it if `initial` isn't specified, or doesn't return a value, it is also used if the user sets the value to empty.